.codecov.yml

```diff
@@ -5,5 +5,6 @@ codecov:
   notify:
     after_n_builds: 12
     wait_for_ci: true
+  token: 2b8d4d69-6de6-4e1d-840a-5ccf9d849565
```

Comment:
How about a comment about why it's okay to include the token here?

Reply:
Added comment.